getting_started/12_flexibility.ipynb

```diff
@@ -0,0 +1,113 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Flexibility and Shedding\n",
+    "By default, an episode ends if one of the following conditions is met:\n",
+    "1. **End of Chronics**: The agent reaches the end of the chronics (timeseries) for an Episode (no more data => scenario ends)\n",
+    "2. **Disconnected Generator / Load**: A load, generator is disconnected from the grid. This also applies to Energy Storage Systems (ESSs) if they are being controlled (agent told it to charge / discharge some energy)\n",
+    "3. **Islanding**: The grid splits into multiple parts / islands. For example, a load of generator is alone on a busbar. \n",
+    "4. **Powerflow Divergence**: The redispatching routine \"diverges\": this means that Grid2OP is not able to find a setpoint for the generators for which both \"pmin, pmax and the ramps constrained\" as well as the \"load / generator balancing the solver \"diverges\" and is not able to compute the next state (often meaning there are no feasible solutions to the Kirchoff's laws)\n",
```

Comment:
I'll try to update the doc but actually point 4 above is two different things:

- either the backend diverge (cannot compute the flows, no feasible solution found)
- or the "grid2op routine" to compute the "redispatching" fails

It is two distinct things.